src/ripple/protocol/impl/PublicKey.cpp

```diff
@@ -221,6 +221,8 @@ verifyDigest(
     Slice const& sig,
     bool mustBeFullyCanonical) noexcept
 {
+    if (sig.size() == 0)
+        return false;
```

Comment:
I don't think this added check is needed as this is taken care of by `ecdsaCanonicality` below.

Reply:
Agreed, I'll remove it.